ios/Capacitor.podspec

```diff
@@ -11,6 +11,8 @@ Pod::Spec.new do |s|
   s.authors = { 'Ionic Team' => 'hi@ionicframework.com' }
   s.source = { :git => 'https://github.com/ionic-team/capacitor.git', :tag => s.version.to_s }
   s.source_files = 'Capacitor/Capacitor/*.{swift,h,m}', 'Capacitor/Capacitor/Plugins/*.{swift,h,m}', 'Capacitor/Capacitor/Plugins/**/*.{swift,h,m}'
+  s.exclude_files = 'Capacitor/Capacitor/Plugins/DefaultPlugins.h'
```

Comment:
It's excluded so the cocoapods build matches. Otherwise cocoapods will try to add it to the public headers but that will generate a warning since it's not included in the umbrella header. But, yes, it's probably easier to just remove it since it is empty.